webpack/scenes/Repos/index.js

```diff
@@ -118,17 +84,27 @@ class RedHatRepositoriesPage extends Component {
   }
 }
 
-// const mapStateToProps = (state) => {
-//   const props = {
-//     redHatRepositoriesResponse: state.redHatRepositories,
-//     redHatRepositorySetsResponse: state.redHatRepositorySets
-//   };
-//   return props;
-// };
-//
-// export default connect(mapStateToProps, {
-//   loadRedHatRepositories,
-//   loadRedHatRepositorySets
-// })(RedHatRepositoriesPage);
-
-export default RedHatRepositoriesPage;
\ No newline at end of file
+RedHatRepositoriesPage.defaultProps = {
+  rhRepos: {},
+  rhRepoSetsResponse: {},
+};
+
+RedHatRepositoriesPage.propTypes = {
+  loadRedHatRepositories: PropTypes.func.isRequired,
+  rhRepos: PropTypes.shape({}),
+  rhRepoSetsResponse: PropTypes.shape({}),
+};
+
+const mapStateToProps = (state) => {
+  const props = {
+    rhRepos: state.redHatRepositories,
+    rhRepoSetsResponse: state.redHatRepositorySets,
+  };
+  return props;
+};
+
+export default connect(mapStateToProps, {
+  loadRedHatRepositories,
+})(RedHatRepositoriesPage);
+
+// export default RedHatRepositoriesPage;
```

Comment:
You can remove this line, I had put that in there in order to easily load the page without errors when redux wasn't connected.

Reply:
Thanks for the feedback on this page! I'll get this removed.